### PR TITLE
Fixes #1182, #1183, #1184, #1185: Concept Card - Tablet

### LINKED ISSUE #1182: Concept Card - Tablet (Landscape) (Lowfi)
Mocks: https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/screen/883dee89-a1cb-4932-9c7d-ab7fb585238c/L-Concept-Card

Implement low-fi UI for **Concept Card** tablet landscape mode

**Target PR date**: 5 July 2020
**Target completion date**: 9 July 2020

### LINKED ISSUE #1183: Concept Card - Tablet (Landscape) (Highfi)
Mocks: https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/screen/883dee89-a1cb-4932-9c7d-ab7fb585238c/L-Concept-Card

Implement high-fi UI for **Concept Card** tablet landscape mode

**Target PR date**: 15 July 2020
**Target completion date**: 19 July 2020

### LINKED ISSUE #1184: Concept Card - Tablet (Portrait) (Lowfi)
Mocks: https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/screen/203c578c-57da-461a-9d4d-60fed26f9b64/P-Concept-Card

Implement low-fi UI for **Concept Card** tablet portrait mode

**Target PR date**: 5 July 2020
**Target completion date**: 9 July 2020

### LINKED ISSUE #1185: Concept Card - Tablet (Portrait) (Highfi)
Mocks: https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/screen/203c578c-57da-461a-9d4d-60fed26f9b64/P-Concept-Card

Implement high-fi UI for **Concept Card** tablet portrait mode

**Target PR date**: 15 July 2020
**Target completion date**: 19 July 2020

### PR DESCRIPTION
**Pull request by MohamedMedhat1998**
## Explanation

- Implemented high-fi UI for **ConceptCard** for both **landscape** and **portrait** mode
- Mocks link (portrait): https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/screen/203c578c-57da-461a-9d4d-60fed26f9b64/P-Concept-Card
- Mocks link (landscape): https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/screen/883dee89-a1cb-4932-9c7d-ab7fb585238c/L-Concept-Card
- Fixes #1182 
- Fixes #1183 
- Fixes #1184 
- Fixes #1185 
### 1024x768 Emulator
screenshot-2020-06-25_14 04 30 23

screenshot-2020-06-25_13 43 58 94
### 2560x1800 Emulator
screenshot-2020-06-25_14 23 09 709

screenshot-2020-06-25_14 22 48 774

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/main/res/layout-sw600dp-land/concept_card_fragment.xml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+      app:layout_constraintBottom_toBottomOf="parent"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toBottomOf="@+id/concept_card_toolbar">
+
+      <ScrollView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:overScrollMode="never"
+        android:paddingBottom="112dp"
```
Comment (rt4914) [inline app/src/main/res/layout-sw600dp-land/concept_card_fragment.xml:42]: I think this should be `138` as per mocks.

File: app/src/main/res/layout-sw600dp-port/concept_card_fragment.xml [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+      app:layout_constraintBottom_toBottomOf="parent"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toBottomOf="@+id/concept_card_toolbar">
+
+      <ScrollView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:overScrollMode="never"
+        android:paddingBottom="112dp"
```
Comment (rt4914) [inline app/src/main/res/layout-sw600dp-port/concept_card_fragment.xml:42]: Here follow padding which you have followed in other mocks.

**Review by rt4914 — APPROVED** [review]
LGTM, address few comments.

File: app/src/main/res/layout-sw600dp-land/concept_card_fragment.xml (hunk 1 above)
Comment (MohamedMedhat1998) [reply app/src/main/res/layout-sw600dp-land/concept_card_fragment.xml:42]: We had a discussion on the paddings in scrollable screens and we decided that the margin (or the padding) between the bottommost item and the bottom of the screen should be 160 including the android bar whose height is 48.  
So, we should use 112 in the code (160 - 48).  
@rt4914 PTAL

File: app/src/main/res/layout-sw600dp-port/concept_card_fragment.xml (hunk 2 above)
Comment (MohamedMedhat1998) [reply app/src/main/res/layout-sw600dp-port/concept_card_fragment.xml:42]: We had a discussion on the paddings in scrollable screens and we decided that the margin (or the padding) between the bottommost item and the bottom of the screen should be 160 including the android bar whose height is 48.  
So, we should use 112 in the code (160 - 48).  
@rt4914 PTAL

Comment (rt4914) [reply app/src/main/res/layout-sw600dp-port/concept_card_fragment.xml:42]: sounds good

**Pull request merged**